assets/css/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #d9dcd6;
+}
+
+/* class selector for a name of class header  */
```

Comment:
Great to see you adding comments here. The comments are actually just to tell us what section of the HTML is being styled, so something like `header styling` would be fine. This applies to the rest of the document too. Try to get rid of comments telling us what you've done or explaining the type of selector, and tell us what they're targeting in the HTML